app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -828,16 +837,17 @@ public void onClick(@NonNull MaterialDialog materialDialog,
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
         inNightMode = SettingValues.isNight();
         disableSwipeBackLayout();
-        super.onCreate(savedInstanceState);
-        if ((getIntent().getFlags() & Intent.FLAG_ACTIVITY_BROUGHT_TO_FRONT) != 0) {
-            // Activity was brought to front and not created
-            finish();
-            return;
-        }
-        if (!Slide.hasStarted) {
-            Slide.hasStarted = true;
```

Comment:
Can you explain your changes here? Why did you move the super call up? Why was this stuff removed?

Reply:
I was trying to figure out it's use, so tried to remove and observe. I was supposed to restore it before committing the changes....Looks like I forgot to do that...Sorry!!